eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -261,3 +271,13 @@ makeGovAction GovernanceActionInfo {..} =
     , gasReturnAddr = gaiRewardAddress
     , gasAction = gaiAction
     }
+
+instance EraPParams era => ToCBOR (ConwayTallyState era) where
+  toCBOR = undefined
```

Comment:
was this somehow not needed before this PR, so that's why you just stubbed it? (just curious, stubbing is fine)

Reply:
Yes, I stubbed it. I made a whole bunch of constraints required for `GovernanceState`: 

https://github.com/input-output-hk/cardano-ledger/blob/e7a76b1d0a6c4800815b76cc5129fc62cff9da14/eras/shelley/impl/src/Cardano/Ledger/Shelley/Governance.hs#L25-L35

Reply:
Both PPUPState and TallyState are used in LedgerState, and all of the above constraints are needed there. Does this answer the question?

Reply:
yep, thanks!